docs/source/user_guide/automl.ipynb

```diff
@@ -148,6 +148,47 @@
     "automl_custom = evalml.automl.AutoMLSearch(problem_type='multiclass', allowed_pipelines=[CustomMulticlassClassificationPipeline])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Limiting the Hyperparameter Search Space\n",
+    "The AutoML search algorithm first trains each component in the pipeline with their default values. After the first iteration, it then tweaks the parameters of these components using the pre-defined hyperparameter ranges that these components have. To limit the search over certain hyperparameter ranges, use `make_pipeline` to define a pipeline with a custom hyperparameter range. Hyperparameter ranges can be found through the [API reference](https://evalml.alteryx.com/en/stable/api_reference.html) for each component. Note: The default value of the component must be included in any specified hyperparameter range for AutoMLSearch to succeed. Additionally, the parameter value must be specified as a list, even for just one value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml import AutoMLSearch\n",
+    "from evalml.demos import load_fraud\n",
+    "from evalml.pipelines.components.utils import get_estimators\n",
+    "from evalml.model_family import ModelFamily\n",
+    "from evalml.pipelines.utils import make_pipeline\n",
+    "\n",
+    "X, y = load_fraud()\n",
+    "# limit the numeric impute strategy to include only `mean` and `median`\n",
+    "# `mean` is the default value for this argument, and it needs to be included in the specified hyperparameter range\n",
+    "custom_hyperparameters = {'Imputer': {\n",
+    "    'numeric_impute_strategy': ['mean', 'median']\n",
+    "}}\n",
+    "\n",
+    "# example of setting parameter to just one value\n",
+    "\"\"\"\n",
+    "custom_hyperparameters = {'Imputer': {\n",
+    "    'numeric_impute_strategy': ['mean']\n",
+    "}}\n",
+    "\"\"\"\n",
```

Comment:
I think we should include this code un-commented. We could also delete it, but I like that you've included this example too.

Reply:
If you want the multi-value hyperparams to be the ones used in your search code below, then please move this definition to the top, so that the multi-value is defined last and is used in the search